File: src/helloWorld.jl
numThreads = Threads.nthreads();
Threads.@threads for i in 1:numThreads
    println("Hello World!"*
"This is thread # $(Threads.threadid())");
end

# Hello World! This is thread # 1
# Hello World! This is thread # 6
# Hello World! This is thread # 3
# Hello World! This is thread # 4
# Hello World! This is thread # 5
# Hello World! This is thread # 2

File: src/helloWorld.py
import threading;
from time import sleep;
import numpy as np;

def helloWorld():
    sleep(np.random.random());
    print("Hello world! This is {}"
          .format(threading.current_thread().name))

if __name__ == "__main__":
    for i in range(6):
        t = threading.Thread(
            target=helloWorld,args=[]);
        t.start()

# Hello world! This is Thread-4
# Hello world! This is Thread-1
# Hello world! This is Thread-6
# Hello world! This is Thread-2
# Hello world! This is Thread-5
# Hello world! This is Thread-3
